.vscode/settings.json

```
@@ -18,5 +18,7 @@
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
+
+    "mochaExplorer.files": "test/*.ts"
```

```suggestion
    "mocha.files.glob": "test/**/*-test.ts",
    "mocha.requires": [
        "ts-node/register"
    ]
```
This fixed the sidebar for me